mock-client/tokens.ts

```diff
@@ -43,4 +49,32 @@ export class TokensController extends BaseController {
     ) as PaginatedTokenResponse;
     return response;
   }
+
+  @Post('/:tokenId/authorize')
+  @ResponseSchema(PaginatedTokenResponse, {
+    statusCode: HttpStatus.OK,
+    description: 'Successful response',
+    examples: {
+      success: TOKENS_LIST_MOCK,
+    },
+  })
+  async authorizeToken(): Promise<AuthorizationInfoResponse> {
+    const ocpiToken = new OcpiToken();
```

Comment:
will there be a todo to make these values dynamic or is this just a testing endpoint that can be removed? :O

Reply:
this is one of the mock endpoints that we use for testing... I am not sure that we want to remove it, I think it can be helpful in testing, but yes, we can technically just have an OpenAPI spec and use `mockoon` to mock it, but since they havent published OpenAPI for v2.2.1, this allows us to write a controller that will kill two birds in one stone in that it helps generate the spec while also performing the mock... You can even layer in `mockoon` to then run using the spec that this `mock-client` produces... I suppose another way of doing it would be to just write out the `json` spec 🤔